src/Components/Components/src/ComponentBase.cs

```diff
@@ -141,7 +146,8 @@ protected virtual void OnAfterRender(bool firstRender)
     }
 
     /// <summary>
-    /// Method invoked after each time the component has been rendered. Note that the component does
+    /// Method invoked after each time the component has been rendered interactively. This method is not
+    /// invoked during prerendering or server-side rendering. Note that the component does
     /// not automatically re-render after the completion of any returned <see cref="Task"/>, because
     /// that would cause an infinite render loop.
```

Comment:
```suggestion
    /// Method invoked after each time the component has been rendered interactively and the UI has finished
    /// updating (for example, after elements have been added to the browser DOM). Any <see cref="ElementReference" />
    /// fields will be populated by the time this runs.
    ///
    /// This method is not invoked during prerendering or server-side rendering, because those processes
    /// are not attached to any live browser DOM and are already complete before the DOM is updated.
    ///
    /// Note that the component does not automatically re-render after the completion of any returned <see cref="Task"/>,
    /// because that would cause an infinite render loop.
```